sdk/core/azure-core/azure/core/pipeline/policies/universal.py

```diff
@@ -266,10 +266,9 @@ def on_response(self, request, response):
 class ContentDecodePolicy(SansIOHTTPPolicy):
     """Policy for decoding unstreamed response content.
     """
-    JSON_MIMETYPES = [
-        'application/json',
-        'text/json' # Because we're open minded people...
-    ]
+    # Accept "text" because we're open minded people...
+    JSON_REGEXP = re.compile(r'^(application|text)/([a-z+.]+\+)?json$')
```

Comment:
Do we know 100% that there will never be digits? I've definitely encountered custom mime types with e.g. `"v1"` in them in some contexts: https://jupyter.readthedocs.io/en/latest/reference/mimetype.html I am not sure how common it is in general, or if there is some basis for extra assumptions here.

Reply:
I'm fine to add digit.

Reply:
Fixed